content/version/1/4/code-of-conduct.id.md

```diff
@@ -0,0 +1,68 @@
++++
+version = "1.4"
+aliases = ["/version/1/4"]
++++
+
+# Kode Etik Contributor Covenant
+
+## Janji Kami
+
+Dalam minat kami untuk membuat lingkungan yang terbuka dan ramah, kami sebagai 
+kontributor dan pengelola berjanji untuk membuat partisipasi dalam proyek kami
+dan komunitas kami bebas dari pelecehan untuk semua orang, tanap memandang usia,
+ukuran tubuh, disabilitas, etnik, identitas gender dan ekspresinya, tingkat pengalaman, pendidikan, status sosial ekonomi, nationalitas, penampilan pribadi, ras, agama, atau identitas dan orientasi seksual.
+
+## Standar Kami
+
+Contoh perilaku yang berkontribusi untuk menciptakan sebuah lingkungan
+yang positif meliputi :
+
+* Memakai bahasa yang ramah dan inklusif
+* Menghormati sudut pandang dan pengalaman yang berbeda
+* Menerima kritik konstruktif dengan baik
+* Fokus terhadap apa yang terbaik bagi komunitas
+* Menunjukan empati kepada anggota komunitas yang lain
+
+Contoh perilaku yang tidak dapat diterima oleh partisipan meliputi :
+
+* Penggunaan bahasa atau citra seksual dan perhatian seksual yang tidak diinginkan.
+* Trolling, komentar menghina, dan serangan terhadap individual atau pandangan politik.
+* Pelecahan umum atau pribadi
+* Menerbitkan informasi pribadi orang lain, seperti alamat fisik atau elektronik,
+tanpa izin.
+* Perilaku lain yang dapat dianggap tidak sesuai dalam lingkungan profesional
+
+## Tanggung Jawab Kami
+
+Pengelola proyek bertanggung jawab untuk mengklarifikasi standar perilaku
+yang dapat diterima dan diharapkan untuk melakukan tindakan korektif yang tepat
+dan adil di Indonesia menanggapi kasus perilaku yang tidak dapat diterima.
+
+Pengelola proyek mempunyai hak dan responsibilitas untuk menghapus, mengedit atau
```

Comment:
'tanggung jawab' instead of 'responsibilitas', please. KBBI has no entry on that.